test/functional/api/extmark_spec.lua

```diff
@@ -1474,7 +1474,7 @@ describe('API/extmarks', function()
   it('in prompt buffer', function()
     feed('dd')
     local id = set_extmark(ns, marks[1], 0, 0, {})
-    curbufmeths.set_option('buftype', 'prompt')
+    curbufmeths.set_option_value('buftype', 'prompt', {})
```

Comment:
```suggestion
    meths.set_option_value('buftype', 'prompt', {buf=0})
```

Reply:
Oh yeah, it seems I did the `curbufmeths` stuff wrong. Will need to fix that